src/client/src/Stream.c

```diff
@@ -3245,3 +3246,46 @@ STATUS resetStream(PKinesisVideoStream pKinesisVideoStream)
     return retStatus;
 }
 
+VOID logStreamInfo(PStreamInfo pStreamInfo) {
+    PCHAR streamingType;
+
+    if (pStreamInfo == NULL) {
+        return;
+    }
+
+    switch (pStreamInfo->streamCaps.streamingType) {
+        case STREAMING_TYPE_REALTIME:
+            streamingType = (PCHAR) "REALTIME";
+            break;
+        case STREAMING_TYPE_OFFLINE:
+            streamingType = (PCHAR) "OFFLINE";
+            break;
+        case STREAMING_TYPE_NEAR_REALTIME:
+            streamingType = (PCHAR) "NEAR_REALTIME";
+            break;
+    }
+
+
+    DLOGD("Kinesis Video Stream Info:\n\t"
+                  "Stream name: %s\n\t"
+                  "Streaming type: %s\n\t"
+                  "Max latency (100ns): %" PRIu64 "\n\t"
+                  "Fragment duration (100ns): %" PRIu64 "\n\t"
+                  "Key frame fragmentation: %s\n\t"
+                  "Absolute frame timecode: %s\n\t"
+                  "Buffer duration (100ns): %" PRIu64 "\n\t"
+                  "Replay duration (100ns): %" PRIu64 "\n\t"
+                  "Connection Staleness duration (100ns): %" PRIu64 "\n\t"
+                  "Frame order mode: %u",
+          pStreamInfo->name,
+          streamingType,
+          pStreamInfo->streamCaps.maxLatency,
+          pStreamInfo->streamCaps.fragmentDuration ? "Yes" : "No",
+          pStreamInfo->streamCaps.keyFrameFragmentation ? "Yes" : "No",
+          pStreamInfo->streamCaps.absoluteFragmentTimes ? "Yes" : "No",
+          pStreamInfo->streamCaps.bufferDuration,
+          pStreamInfo->streamCaps.replayDuration,
+          pStreamInfo->streamCaps.connectionStalenessDuration,
+          pStreamInfo->streamCaps.frameOrderingMode);
```

Comment:
I think you should dump out some of the other fields that might be important
contentType,
frameTimecodes,
nalAdaptationFlags,
avgBandwidthBps, --> this and the next one would give some idea what the size of the view would be but we already print out the size
frameRate,
replayDuration,
connectionStalenessDuration,
segmentUuid

Most importantly, we might want to print out also the track infos